test.md

```diff
@@ -0,0 +1 @@
+"This file should not be considered"
```

Comment:
add test.m to the .giignore file

Reply:
- [ ] This reads like an order, kindly change it to be a polite suggestion, also explain why the change needs to be done. e.g. (Kindly add test.md to the .gitignore file, so it doesn't get pushed to the repository.)

Reply:
The test.md file should not be on the remote repository. Kindly add it to the .gitignore file.